### ISSUE TITLE
After clicking "Print" to print the labels there should be a way to edit the labels before actually printing them to a physical paper label.  This exists in SpineOMatic and Libstick (non cloud app versions)

### ISSUE THREAD

**Issue by YoelKortick** [ExLibrisGroup/spineomatic-cloudapp#2]
This came from McGee, Margaret <Margaret.McGee@suny.edu>
She is seriously testing the SpineOMatic Cloud App and I am working closely with her.

After clicking "Print" to print the labels there should be a way to edit the labels before actually printing them to a physical paper label.  This exists in SpineOMatic and Libstick (non cloud app versions)

Sometimes content is added, or the institution wants to split a line, etc.

**Comment by SWITCHconsortium** [comment]
Yes, please. We almost have the Cloud App printing as desired, but we are not willing to make the move to the cloud app without the ability to edit the labels before printing. We do not need to do it too often, but when we do, it is critical.

**Comment by VangelisPalaskas** [comment]
Dear Josh,
I think that Yoel’s suggestion would be quite useful for KU Leuven Libraries. The main use case is for some ‘Physical Titles’ in Alma that contain the same Call Number for two or more Physical Items (usually it refers to different volumes). 

In the example below, the same Call Number refers to two separate physical items and hence two separate labels (i.e. BUSE vol.1 & vol.2). Practically, we wish to edit the Call Number before printing the label to restrict the printed value only to the correspondent physical item (i.e. vol.1 or vol.2).

We understand if it takes some time to implement this but would ‘editing before printing’ be something that you are looking into?

<img width="472" alt="Editing before printing 2" src="https://user-images.githubusercontent.com/89778038/141458282-8cb8e285-67c3-4264-8816-1050d76ae7c9.png">

Indeed this feature is in our current desktop app but we look into switching to SpineOMatic Cloud App anyhow. It’s the future! 

Thank you in advance.

Kind regards,

Vangelis Palaskas.

**Comment by jweisman** [comment]
Hi @VangelisPalaskas - thanks for your feedback. This enhancement request is a bit complex to implement. We're still considering how it can be done.

**Comment by VangelisPalaskas** [comment]
Hi @jweisman. I hope that you are well. 

Although we have rolled-out SpineOMatic to a number of our partners (e.g. Flemish Parliament and National Bank of Belgium) and many of them are extremely happy with its user-friendliness and ease-of-use, the feature 'edit before printing' (mainly for the Call Numbers) is still very important to implement it within KU Leuven.

Are there any news on that? If not, is there any way that I can contribute to escalate/facilitate this enhancement?

I hope that I am not too much about it but the enhancement is critical because the alternative solution would entail creating hundreds of new holdings for existing items. Thus, it would be very helpful to have this feature.

Kind regards,

Vangelis Palaskas.

**Comment by mgobat** [comment]
@VangelisPalaskas, we don't yet have an estimate for when this feature will be available.

**Comment by VangelisPalaskas** [comment]
@mgobat Thank you for the answer. I understand. 

However, I need to ask: Is it something that Ex Libris has put into development but it is not yet ready or it is possible that the feature will not be developed?

**Comment by mgobat** [comment]
@VangelisPalaskas, we understand that this feature is highly desired by a number of Alma users, We are still evaluating options for ways to implement this enhancement.  The feature may be added depending on our options for implementation. Sorry I can't be more definitive right now.

**Comment by VangelisPalaskas** [comment]
Just dropping this here as an idea. I know that it is not in the plans yet. 

This is how users edit the Call Number before printing the label in the old label printing tool 📑:

![image](https://user-images.githubusercontent.com/89778038/177164032-bb3a0214-eb03-4e77-a5de-63a670e16fc3.png)

**Comment by simonreadnz** [comment]
Otago supports this enhancement too. We'd really like this to be in place before considering moving form the Desktop app to the Cloud app

**Comment by mgobat** [comment]
Preview/editing labels before printing available in the v1.4.0-beta version. To try the beta, click the "i" (information) icon on the SpineOMatic tile in the "Cloud App Center" in Alma, then click the "Try v1.4.0-beta" link.

Please add feedback/comments to this issue. Thank you.
